hack/testdata/rollingupdate-statefulset-rv2.yaml

```diff
@@ -39,3 +26,12 @@ spec:
         - sh
         - -c
         - 'while true; do sleep 1; done'
+      - name: nginx-2
+        image: gcr.io/google_containers/nginx-slim:0.8
```

Comment:
Would you update `nginx` to `gcr.io/google_containers/nginx-slim:0.8` and add another container (e.g. `gcr.io/google-containers/pause:2.0`).